samples/modelbuilder/BinaryClassification_Sentiment_Razor/SentimentRazorML.Model/SentimentRazorML.Model.csproj

```diff
@@ -1,10 +1,10 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
Were we expecting this?

Reply:
Yes. I'm not sure why it wasn't the case in the original sample. The `SentimentRazorML.Model` class library is supposed to be .NET Standard.